.env-template

```diff
@@ -0,0 +1,12 @@
+LOCAL_STORAGE = <Bool> # Specifies whether to use database or inmem proejct handler
+DB_CONNECTION_STRING= <conntection-string-for-the-database>
+AUTH_SECRET_KEY = <key-for-used-for-generating-authentication-jwts>
+JOIN_SECRET_KEY = <key-for-used-for-generating-tokens-for-project-join>
+SES_SENDER = <email-address-registered-with-AWS-SES-for-sending-emails>
+AWS_ACCESS_KEY_ID = 
+AWS_SECRET_ACCESS_KEY = 
+AWS_DEFAULT_REGION = 
+RAW_LOGO_BUCKET=<s3-bucket-name>
+RESIZED_LOGO_BUCKET=<s3-bucket-name>
+DOCUMENTS_BUCKET=<s3-bucket-name>

```

Comment:
It's more as a hint then change request. If you know that your application would not have terabytes of data, you can use one bucket.
Split of the data could be done by key prefix - documents/*, raw_logo/*, resized_logo/*
As far as I remember, you can set up permissions to data by other resources with prefixes

It's just IMHO, that you have more observability if you store everyth in one place.

Reply:
Thanks for the advice. I agree with your points. However, since you don't consider it a crucial change and it is not a trivial thing to re-implement so near to the deadline, I will leave it as it is right now. 
In any case, thanks for the advice and I will definitely keep it in mind for the future :)